### PR TITLE
completions steps were disconnected from algo, and 'append' instruction was phrased as a 'join'

### LINKED ISSUE #142: AccName "completion" step is disconnected from the computation list
This text comes after "Step 2" in the AccName computation, but it's not part of the computation list. Should it be? I was thinking this could be "Step 3" or the named step "Completion." 

> Append the result of each step above, with a space, to the total accumulated text.
> After all steps are completed, the total accumulated text is used as the accessible name or accessible description of the element that initiated the computation.

### PR DESCRIPTION
**Pull request by cookiecrook**
closes #142

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/accname/pull/187.html" title="Last updated on Feb 7, 2023, 9:57 PM UTC (ec7a9a2)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/accname/187/9c71070...ec7a9a2.html" title="Last updated on Feb 7, 2023, 9:57 PM UTC (ec7a9a2)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -363,7 +363,7 @@ <h4>Computation steps</h4>
                       <ol>
                         <li id="comp_labelledby_set_current" name="step2B.ii.a">Set the <code>current node</code> to the node referenced by the IDREF.</li>
                         <li id="comp_labelledby_recursion" name="step2B.ii.b"><em>LabelledBy Recursion:</em> Compute the text alternative of the <code>current node</code> beginning with the overall <a href="#comp_computation">Computation</a> step. Set the <code>result</code> to that text alternative.</li>
-                        <li id="comp_labelledby_append" name="step2B.ii.c">Append the <code>result</code>, with a space, to the <code>accumulated text</code>.</li>
+                        <li id="comp_labelledby_append" name="step2B.ii.c">Append a space character and the <code>result</code> to the <code>accumulated text</code>.</li>
```
Comment (cookiecrook) [inline index.html:366]: Justification for including this change: while adding `id="comp_append"` below, I noticed these "append" steps were actually phrased as "joins" ("append the result with a space" is `+= "result "` not `+= " result"`) so I rephrased for accuracy. If the editors prefer, I can include this separately.

File: index.html [hunk 2]
```diff
@@ -464,11 +464,11 @@ <h4>Computation steps</h4>
                 <p>Tooltip attributes are used only if nothing else, including subtree content, has provided results. </p>
               </details></div>
             </li>
+            <li id="comp_append">Append a space character and the <code>result</code> of each step above to the <code>total accumulated text</code>.</li>
```
Comment (cookiecrook) [inline index.html:467]: same here…

Comment (cookiecrook) [reply index.html:467]: Still editorial IMO.

File: index.html (hunk 1 above)
Comment (MelSumner) [reply index.html:366]: I think it's fine to leave in here, it's essentially equivalent to "fixing a typo" IMO

**Review by MelSumner — APPROVED** [review]
👍 :shipit:

**Comment by cookiecrook** [comment]
@jnurthen @spectranaut merge?

**Review by accdc — APPROVED** [review]

**Pull request merged**